operations/System_Recovery/Slurm_Service_Recovery.md

```diff
@@ -10,10 +10,10 @@ The following covers restoring Slurm data.
 
 ## Service recovery for Slurm
 
-To restore Slurm data from backup, follow sections
-*10.3.11 Restore Slurm Accounting Database from Backup* and
-*10.3.13 Restore Slurm Spool Directory from Backup* in the
-**HPE Cray Programming Environment Installation Guide: CSM on HPE Cray EX Systems (S-8003)**.
+To restore Slurm data from backup, see
+*Restore Slurm accounting database from backup* and
+*Restore Slurm spool directory from backup* in the
+**HPE Slurm Installation Guide: CSM on HPE Cray EX Systems (S-8058)**.
```

Comment:
```suggestion
To restore Slurm data from a backup, see
*Restore Slurm accounting database from a backup* and
*Restore Slurm spool directory from a backup* in the
**HPE Slurm Installation Guide: CSM on HPE Cray EX Systems (S-8058)**.
```

Reply:
Added a few minor changes from the CSM PRs I'm working on.

Reply:
I'm basing that string on the section name in the actual doc, so I think leaving it w/o the added " a " is best.